app/feeswapper/management/commands/feeSwapper.py

```diff
@@ -0,0 +1,185 @@
+import json
+import time
+
+from django.conf import settings
+from django.core.management.base import BaseCommand, CommandError
+from django.utils.timezone import now
+
+import requests
+import sendgrid
+from feeswapper.models import CurrencyConversion
+from sendgrid.helpers.mail import Content, Email, Mail
+from web3 import Web3
+from web3.middleware import geth_poa_middleware
+
+# Amount of slippage from target Ether price from estimated price on exchange allowed when trading tokens back to ETH
+SLIPPAGE = 0.05  
+
+# ABIs needed for Uniswap interactions
+factoryABI = """[{"name": "NewExchange", "inputs": [{"type": "address", "name": "token", "indexed": true}, {"type": "address", "name": "exchange", "indexed": true}], "anonymous": false, "type": "event"}, {"name": "initializeFactory", "outputs": [], "inputs": [{"type": "address", "name": "template"}], "constant": false, "payable": false, "type": "function", "gas": 35725}, {"name": "createExchange", "outputs": [{"type": "address", "name": "out"}], "inputs": [{"type": "address", "name": "token"}], "constant": false, "payable": false, "type": "function", "gas": 187911}, {"name": "getExchange", "outputs": [{"type": "address", "name": "out"}], "inputs": [{"type": "address", "name": "token"}], "constant": true, "payable": false, "type": "function", "gas": 715}, {"name": "getToken", "outputs": [{"type": "address", "name": "out"}], "inputs": [{"type": "address", "name": "exchange"}], "constant": true, "payable": false, "type": "function", "gas": 745}, {"name": "getTokenWithId", "outputs": [{"type": "address", "name": "out"}], "inputs": [{"type": "uint256", "name": "token_id"}], "constant": true, "payable": false, "type": "function", "gas": 736}, {"name": "exchangeTemplate", "outputs": [{"type": "address", "name": "out"}], "inputs": [], "constant": true, "payable": false, "type": "function", "gas": 633}, {"name": "tokenCount", "outputs": [{"type": "uint256", "name": "out"}], "inputs": [], "constant": true, "payable": false, "type": "function", "gas": 663}]"""
```

Comment:
Lets put these in a separate file

Reply:
Done locally.  Will include in next commit.